src/Application/Handlers/HttpErrorHandler.php

```diff
@@ -73,7 +73,6 @@ protected function respond(): Response
 
         if (
             !($exception instanceof HttpException)
-            && ($exception instanceof Exception || $exception instanceof Throwable)
```

Comment:
this could never have been false

Reply:
Yep, this is the one I tweaked while porting to Identtiy